File: DemoApp/DemoAppJava/app/src/main/java/co/optable/demoappjava/TheApplication.java
package co.optable.demoappjava;

import android.app.Application;
import co.optable.sdk.OptableConfig;
import co.optable.sdk.OptableSDK;

public class TheApplication extends Application {

    public static OptableSDK optable;

    @Override
    public void onCreate() {
        super.onCreate();

        OptableConfig config = new OptableConfig(this, "prebidtest", "android-sdk");
        config.setOrigin("https://www.optable.co");
        optable = new OptableSDK(config);
    }

}

File: DemoApp/DemoAppKotlin/app/src/main/java/co/optable/androidsdkdemo/TheApplication.kt
package co.optable.androidsdkdemo

import android.app.Application
import co.optable.sdk.OptableConfig
import co.optable.sdk.OptableSDK

class TheApplication : Application() {

    companion object {
        lateinit var optable: OptableSDK
    }

    override fun onCreate() {
        super.onCreate()

        val config = OptableConfig(
            providedContext = this,
            tenant = "prebidtest",
            originSlug = "android-sdk",
            host = "ca.edge.optable.co",
            origin = "https://www.optable.co",
        )
        optable = OptableSDK(config)
    }

}
